packages/compiler/test/parser.test.ts

```diff
@@ -137,7 +137,6 @@ describe("compiler: parser", () => {
       ["model bar<a, b> = a | b;", [/'{' expected/]],
     ]);
   });
-
```

Comment:
Reverts an unrelated change to parser.test.ts earlier in the PR. I don't actually care about this, it just fell out of how I was reviewing the overall diff of PR + my changes vs. main.